.jshintrc

```diff
@@ -37,8 +37,7 @@
     "boss"          : false,     // true: Tolerate assignments where comparisons would be expected
     "debug"         : false,     // true: Allow debugger statements e.g. browser breakpoints.
     "eqnull"        : false,     // true: Tolerate use of `== null`
-    "es5"           : false,     // true: Allow ES5 syntax (ex: getters and setters)
-    "esnext"        : false,     // true: Allow ES.next (ES6) syntax (ex: `const`)
+    "esversion"     : 6,
```

Comment:
Is this related? You will also need to add a `.jshint` with es5 to `static/js` because we can't support es6 in browsers yet.

Reply:
Yeah - es6 should be allowed in `Gruntfile`.